vhdbuilder/packer/install-dependencies.sh

```diff
@@ -648,6 +648,7 @@ AZUREDISK_CSI_VERSIONS="
 0.7.0
```

Comment:
do we still use this? if not we should remove it to preserve disk space

Reply:
I think it's not needed but @andyzhangx could correct me.

Reply:
yes, let's move this